### PR TITLE
Db4

### PR DESCRIPTION
**Pull request by DuaneDave**
### In this PR:

- the many to many concept was implemented
- New tables were added to the DB
- Updates and insertion was made to tables
- New queries were made

Screenshots
[Screenshot 1](https://github.com/DuaneDave/vet_clinic-DB/blob/db4/screenshots/l1.JPG)
[Screenshot 2](https://github.com/DuaneDave/vet_clinic-DB/blob/db4/screenshots/l2.JPG)
[Screenshot 3](https://github.com/DuaneDave/vet_clinic-DB/blob/db4/screenshots/l3.JPG)
[Screenshot 4](https://github.com/DuaneDave/vet_clinic-DB/blob/db4/screenshots/l4.JPG)
[Screenshot 5](https://github.com/DuaneDave/vet_clinic-DB/blob/db4/screenshots/l5.JPG)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -22,6 +22,24 @@ CREATE TABLE species (
     name varchar(100)
 );
 
+CREATE TABLE vets (
+    id BIGSERIAL PRIMARY KEY,
```
Comment (mikethreels) [inline schema.sql:26]: - [ ] please make sure that the id can't be null.

File: queries.sql [hunk 2]
```diff
@@ -50,4 +50,19 @@ SELECT animals.name, owners.full_name FROM animals RIGHT JOIN owners ON animals.
...
 SELECT animals.name from animals JOIN owners ON owners.id = animals.owner_id WHERE animals.escape_attempts = '0' AND animals.owner_id = '5';
+SELECT full_name, COUNT(owner_id) FROM owners JOIN animals on owners.id = animals.owner_id GROUP BY full_name ORDER BY COUNT (owner_id) desc limit 1;
+
+
+/* Day 4 */
+
+SELECT animals.name FROM animals JOIN visits ON animals.id = visits.animal_id WHERE vet_id = '1' ORDER BY visits.date_of_visit desc limit 1;
+SELECT COUNT(DISTINCT animal_id) FROM visits WHERE vet_id = '2';
+SELECT vets.name, specialization.vet_id, specialization.species_id FROM vets LEFT JOIN specialization ON specialization.vet_id = vets.id;
+SELECT animals.name , visits.date_of_visit FROM animals JOIN visits ON animals.id = visits.animal_id WHERE visits.date_of_visit BETWEEN '2020-04-01' AND '2020-08-30' AND vet_id = '3';
```
Comment (mikethreels) [inline queries.sql]: - [ ] according to the requirements you should List all vets and their specialties, including vets with no specialties as well.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi David, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (DuaneDave) [reply schema.sql:26]: Thank you for the comment.
I would like to remind you that this was not stated in the requirement

```
Create a table named vets with the following columns:
id: integer (set it as autoincremented PRIMARY KEY)
name: string
age: integer
date_of_graduation: date
```

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @DuaneDave ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**